fuzz/src/arb_range.rs

```diff
@@ -50,7 +50,7 @@ impl<T> RangeBounds<T> for ArbRange<T> {
 }
 
 impl<T: Arbitrary> Arbitrary for ArbRange<T> {
```

Comment:
The whole `impl` can be removed in favour of a simple `#[derive(Arbitrary)]`, as soon as a new version of `arbitrary` is released.